guide/05-working-with-the-spatially-enabled-dataframe/part2_data_io_reading_data.ipynb

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
<p>Here, we will subset the data for the state and population class fields and apply a postfix clause.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>